etc/syntax/reference-files/suite_empy.rc

```diff
@@ -0,0 +1,75 @@
+#!empy
+# ^ SYNTAX:3.1
+
+# SYNTAX numbering matches the table in https://github.com/cylc/cylc/issues/2752
+
+# SYNTAX: 3.2
+# templating code should be displayed diffently to cylc code
+@# no comment  # should display as a comment
+@foo
+@{foo="bar"}
+@{ foo = "bar" }
+@{ foo = "bar"; bar = "baz" }
+@{ foo = '%s' % 'bar' }
+@{ set foo = {"1": 2, "3", []} }  # highlight to the last }
+@{ set foo = foo.bar(1, 2, 3) }  # highlight to the last }
+@{ foo_bar }
+@{foo_bar}
+@{ foo.bar(1, 2, 3, []) }  # highlight to the last }
+@[if x]a = b@[end if]  # highlight all but "a = b"
+@[if x.y[]]a = b@[end if]  # highlight all but "a = b"
+a = b  
+# SYNTAX 3.3
+# non-exhaustive list of advanced empy syntax
+@# no comment
+@?NewContextName
+@!100
+invisible@ whitespace  # the @ character should be highlighted
+foo = @\0 bar @\xHH # @\0 and @\xHH should both be highlighted
+foo = @@  # the @@ should NOT be highlighted
+@"foo"  # this should be highlighted the same as @foo?
+@( x ? "true" ! "false )
+@os.environ
+@`x`
+@:2 + 2:
+:%title  "Cylc"
+**'@<CONTENTS>'**  # ???
```

Comment:
I think you meant to update this comment?

Reply:
Updated, still not quite sure what it does!